sys/ztimer/auto_init.c

```diff
@@ -55,7 +55,8 @@
 
 #if MODULE_ZTIMER_USEC
 #  if CONFIG_ZTIMER_USEC_TYPE_PERIPH_TIMER
-static ztimer_periph_timer_t _ztimer_periph_timer_usec = { .min = CONFIG_ZTIMER_USEC_MIN };
+static ztimer_periph_timer_t _ztimer_periph_timer_usec =
+{ .min = CONFIG_ZTIMER_USEC_MIN };
```

Comment:
That looks weird... Wouldn't

```suggestion
static ztimer_periph_timer_t _ztimer_periph_timer_usec = {
        .min = CONFIG_ZTIMER_USEC_MIN
    };
```

Be nicer-looking and more in line with our coding conventions?

Reply:
I leave that to kaspar, in the PR he said he wished to use the automatic output, since it meant less code maintenance for him, so I feel that's the contributor choice. So @kaspar030 id you agree with @miri64 address and squash right away, otherwise you can merge.

Reply:
`uncrustify` should not change and usually produces output as I proposed (see e.g. https://github.com/RIOT-OS/RIOT/pull/13975/files#diff-93334a936917a21c9e73a7273ff6f9b9R71-R75). So this seems more of an `uncrustify` bug to me, that might be fixed in later versions.

Reply:
I agree that the uncrustify version looks weird. When manually splitting the line, uncrustify does this:

```
~  58 static ztimer_periph_timer_t _ztimer_periph_timer_usec = {                                                                   
~  59     .min = CONFIG_ZTIMER_USEC_MIN                                                                                            
+  60 };                                                                                                                           
```

I think it looks better (@miri64's was perfect, but uncrustify doesn't want). I've amended that change.